dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/taglib/GlobalTagLibraryCache.java

```diff
@@ -128,9 +128,9 @@ public GlobalTagLibraryCache(ClassLoader loader) {
             
             TagLibCacheConfigParser tagLibCacheConfigParser = new TagLibCacheConfigParser();
 
-            if(com.ibm.ws.jsp.webcontainerext.JSPExtensionFactory.isPages30orHigher()){
+            if(com.ibm.ws.jsp.PagesVersionHandler.isPages30Loaded()){
```

Comment:
This looks to change the behavior here. What about Pages 3.1? Would this still be relevant code for Pages 3.1?

Reply:
Can you create an issue just as a place holder so we don't forget to do this?

Reply:
https://github.com/OpenLiberty/open-liberty/issues/21414